.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "0.10"
```

Comment:
I'll bring this up and all other issues in meeting today one by one, and then update the PR.